### ISSUE TITLE
Support Schwalbe

### ISSUE THREAD

**Issue by leberwurstsaft** [BenchR267/Get-Schwifty#4]
See https://github.com/kiliankoe/schwalbe

🤣

**Comment by BenchR267** [comment]
Schwalbe is transpiled to Swift, so this should already be possible ;)

**Comment by leberwurstsaft** [comment]
But not yet in your app :)

**Comment by BenchR267** [comment]
True, but there are a lot of more important features that should be implemented before ;)

**Comment by leberwurstsaft** [comment]
![71bf4186f2fd86fe7990f814bb956dd6d77d68ee46e6de9d981f02268d87422f](https://cloud.githubusercontent.com/assets/108899/24721012/3bd8785e-1a3f-11e7-8f83-4257836dfd2a.jpg)
